### ISSUE TITLE
The Missing Diplomat (Quest ID: 1248; The Part to Speak to Mikhail)

### ISSUE THREAD

**Issue by rikamayaz** [chromiecraft/chromiecraft#599]
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [X] enUS
- [ ] other (specify)

##### FACTION

- [X] Alliance
- [ ] Horde

##### CONTENT PHASE:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [X] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

https://wowgaming.altervista.org/aowow/?quest=1248 (The Quest)
https://wowgaming.altervista.org/aowow/?npc=482 (The Quest Starter)
https://wowgaming.altervista.org/aowow/?npc=4963 (The Quest Ender)
https://wotlkdb.com/?npc=4963 (The Quest Ender; The link above is not working at all so I need to use different WOW database)

So, the problem is that I cannot interact with the Quest Ender at all. No yellow question mark or anything when I try to interact with the NPC, although the yellow question mark strangely appeared on the map. The previous part of the quest so far has worked just fine and I am stuck at this stage of the quest

##### EXPECTED BLIZZLIKE BEHAVIOUR:

You can speak to Mikhail to finish the quest and the quest continues to the next stage.

##### STEPS TO REPRODUCE THE PROBLEM:

1. You need to finish the prereq until you arrive at this stage where Eiling Trias asked you to speak to Mikhail at Wetlands
2. I cannot speak to him at all!

##### EXTRA NOTES:

It's quite a straightforward situation. I cannot speak with the Quest Ender to continue to the next part of the chain quest.

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/6947789622de45479d2311fd5d1635af3a957b5d

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by Rittmeister39** [comment]
I actually wanted to report this last week already, but people in chat said it is a known bug, so i didnt bother writing it up.

**Comment by Stump876** [comment]
Can confirm. This bug seems different than the others that were fixed.
At first I was confused. I wasn't sure how someone could fail a redirect quest. But I accepted the follow up quest, 1249, to defeat Tapoke "slim" Jahn, then failed that quest, abandoned it, and I had no option to reaccept. I went on as a different character to try to turn in quest 1248 the intial quest and there was no question mark to indicate a turn in, which I checked and was there before failing quest 1249. Sorry if that sounds confusing. 

Basically someone failing quest 1249 will lock all players from interacting with Mikhail (GUID 9432)

**Comment by Crixxus99** [comment]
I completed this on a couple of characters previously without issue.  Tried on a new character a few days ago and Mikhail was bugged, so I couldn't progress.  I again tried today after a server reset (yesterday?) and was able to get the quest to fight Tapoke "Slim" Jahn.  Won the fight and was told to go back to the inn.  Mikhail was bugged again, so cannot hand in.

**Comment by Rittmeister39** [comment]
same. I subdued Tapoke Slim Jahn, he says he will give info in the Tavern, but he does not react to anything anymore inside.

**Comment by kinggraham72** [comment]
Stuck at this step on this quest right now. Tried last night, and this morning as I type this. I am unable to progress through this quest. Mikhail basically tells me he's busy and to come back later. Well, later never comes.

**Comment by Crixxus99** [comment]
Still bugged.  Can't talk to Mikhail.

**Comment by Eldarix** [comment]
I've been able to advance to the next quest phase and defeat the 'Slim' guy, but again the same happens and the Npc dont give the next:

![WoWScrnShot_060921_124604](https://user-images.githubusercontent.com/82579073/121333824-cfa91300-c921-11eb-886a-9a850153a828.jpg)
![WoWScrnShot_060921_124612](https://user-images.githubusercontent.com/82579073/121333901-e2234c80-c921-11eb-9c17-9d53401c5a32.jpg)

**Comment by fendwe** [comment]
I was able to go through the entire quest chain on the PTR - with one hiccup.
![image](https://user-images.githubusercontent.com/61170874/159978233-f3243794-55b9-407b-9542-5d4a1acf841d.png)

On [part 5 of The Missing Diplomat](https://wowgaming.altervista.org/aowow/?quest=1244) you are supposed to collect [Defias Docket](https://wowgaming.altervista.org/aowow/?item=5947) from [Defias Strongbox](https://wowgaming.altervista.org/aowow/?object=21052). 
The strongbox, however, is non-interactive. I could only proceed with the quest by adding the item manually.

**Comment by FrancescoBorzi** [comment]
it looks like the original AC linked issue has been closed but probably this needs still some attention, marking as [Needs Triage]
